example/src/components/RoomParticipant.tsx

```diff
@@ -0,0 +1,94 @@
+import { BrandColors, AdditionalColors, TextColors } from '@colors';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import { getShortUsername } from '@utils';
+import React from 'react';
+import { View, StyleSheet, ViewStyle } from 'react-native';
+
+import { Icon } from './Icon';
+import { NoCameraView } from './NoCameraView';
+import { Typo } from './Typo';
+
+type RoomParticipantProps = {
+  participant: Membrane.Participant;
+  pStyle: ViewStyle[];
+};
+
+export const RoomParticipant = ({
+  participant,
```

Comment:
we can destructure `participant` here, avoiding repeating `participant.something` all the time

Reply:
Done.